### PR TITLE
updated readme.md overview and added quickstart guide

### PR DESCRIPTION
**Pull request by prasek**
updated readme.md overview and added quickstart guide

[skip ci]

### CODE REVIEW

File: docs/quickstart.md [hunk 1]
```diff
@@ -0,0 +1,947 @@
...
+
+and wait for them to be `Ready:True`
+```
+NAMESPACE           NAME               READY   SOURCE                PACKAGE
+crossplane-system   provider-gcp       True    registry.upbound.io   crossplane/provider-gcp:v0.8.0
+crossplane-system   stack-gcp-sample   True    registry.upbound.io   crossplane/stack-gcp-sample:v0.4.0
+```
+
+#### Create gcp-secret.json
+Run the following replicing `MY_GCP_PROJECT_ID` with your own:
```
Comment (hasheddan) [inline docs/quickstart.md]:
```suggestion
Run the following replacing `MY_GCP_PROJECT_ID` with your own:
```

File: docs/quickstart.md [hunk 2]
```diff
@@ -0,0 +1,947 @@
...
+```
+curl -s https://raw.githubusercontent.com/crossplane/stack-minimal-gcp/master/hack/gcp-credentials.sh | PROJECT_ID=[MY_GCP_PROJECT_ID] bash -
+```
+
+Save the secret JSON in the output to the file `gcp-secret.json` and be sure to delete it later!
+
+#### Generate stack.yaml
+
+Run the following to generate the `stack.yaml` replicing `MY_GCP_PROJECT_ID` with your own:
```
Comment (hasheddan) [inline docs/quickstart.md]:
```suggestion
Run the following to generate the `stack.yaml` replacing `MY_GCP_PROJECT_ID` with your own:
```

**Review by hasheddan — CHANGES_REQUESTED** [review]
A few minor typos, otherwise LGTM!

File: docs/quickstart.md (hunk 1 above)
Comment (prasek) [reply docs/quickstart.md]: fixed

File: docs/quickstart.md (hunk 2 above)
Comment (prasek) [reply docs/quickstart.md]: fixed

**Comment by prasek** [comment]
@hasheddan should be good to go now, thx

**Review by hasheddan — APPROVED** [review]

**Comment by prasek** [comment]
@jbw976 @negz did a final pass and verified everything works as expected, so think this is good to merge, thx

**Review by jbw976 — APPROVED** [review]

**Pull request merged**